ddp-workspace/projects/ddp-dsm-ui/src/app/WELCOME/auth/auth.component.ts

```diff
@@ -43,12 +44,24 @@ export class AuthComponent implements OnInit {
   popUpIsShown = true;
   authError: Observable<string | null>;
   pickList$: Observable<any>;
+  private source$: Observable<StorageEvent>;
 
   constructor(private auth: Auth, private title: Title, private router: Router) {}
 
   ngOnInit(): void {
+
```

Comment:
As this component is in a lazily loaded module and is under the `canLoad` guard, there will be cases (e.g. refreshing tab while logged in) when this component won't be downloaded, therefore this logic about listening to the storage event won't be executed, which means that the user still will need to refresh the other tab to detect the change, even though thanks to guards, navigation won't be allowed anymore, still the user will no longer log out automatically from the application. 

https://user-images.githubusercontent.com/77500504/229788734-812367dc-4512-4c68-8b05-f086771a778e.mov

Reply:
I've moved this to the auth service.  Good catch.